CHANGELOG.md

```diff
@@ -1,3 +1,343 @@
+## Fleet 4.65.0 (Feb 28, 2025)
+
+### Bug fixes
+
+- Fixed UI for window profiles error message being cut off in the OS settings modal
+
+- Added pagination inside each of the Manage Automations modals for policies
+
+- Fixed user page responsiveness to not overflow horizontally in Fleet UI
+
+- Added ability to target app store apps with include/exclude labels in Fleet UI
+- Added ability to edit targets or self-service options for app store apps in Fleet UI
+- Added details modal for add, edit, and delete app store app global activities in Fleet UI
+
+- Fixed policy software automation that falsely reported success when updates actually failed in Fleet UI
+
+- Added clearer error states to metadata-related fields in the SSO settings form
+
+- Fixed case consistency for "Disk encryption" in host OS settings modal
+
+- Ensured consistency of on-click behavior of table rows in Fleet UI
+- Created behavior where clicking on a row of a table directs the user to the details of that row in Fleet UI
+
+- Implemented GitOps Mode
+
+- Removed duplicate Linux lock and wipe scripts from repository
+
+- Prevented an invalid `FLEET_VULNERABILITIES_MAX_CONCURRENCY` value from causing deadlocks during vulnerability processing
+
+- Fixed a bug where uploading a macOS installer could prevent the software from being inventoried
+
+- Fixed a bug where target selector was present in a premature stage
+
+- Fixed a bug that caused macOS App Store apps to show up in Fleet as Windows apps if the Windows version of the app was already in Fleet
+
+- Fixed styling for manage automation buttons and dropdown in Fleet UI
+
+- Surfaced cleaner errors when adding Fleet-maintained apps in Fleet UI
+
+- Hosts that are restored from ABM no longer have old activities in their feed
+
+- Added download URL for Fleet-maintained apps as `url` property on `fleet/software/fleet_maintained_apps/:id`
+
+- Allowed VPP apps to be automatically installed via a Fleet-created policy
+- Added auto-install to FMA via the API, replacing a more brittle client-side implementation
+
+- Fixed a bug where query reports were not being recorded for hosts configured with `--logger_snapshot_event_type=true`
+
+- Added gzip compression for static CSS and JS assets to decrease bundle download times
+
+- Added UI for viewing certificate details on the host details and my device pages
+
+- Added Android MDM activities
+
+- Clarified text on the Policies and Queries pages when no policies/queries exist for the selected team (or All Teams)
+
+- Added UI to the BYOD enrollment page to support enrolling Android devices into Fleet MDM
+
+- Resolved false negatives on vulnerabilities for IntelliJ IDEA Community Edition on Windows
+
+- Changed the copy for the delete and transfer host modal to be more clear about the disk encryption key behavior
+
+- Updated the help text for 3 tabs of the Add hosts modal
+
+- Dropped support for macOS 13.x
```

Comment:
I know you are probably going to reformat this, just wanted to highlight it anyway just in case